### ISSUE TITLE
Format Subpage | References Slide following Introduction

### ISSUE THREAD

**Issue by jrvictor** [compensationServiceTraining/curriculumCreationGuide#39]
213F staff received the following feedback from ISD for the Telephone Development module they created:

- ISD Comment:  References are not included in the course Introduction.
- ISD Recommendation:  Add a slide after the objectives that includes references and standards.

Mary is in agreement that we should consider _identifying relevant references in the course introduction_ as well as at the end on the WYLTKM slide. I am also in agreement with this recommendation and there has been chatter amongst 213F staff that this is particularly important to our VIP curriculum for VSR trainees for the following reasons:

- VSR trainees are typically entry-level external hires, who are not yet familiar with our resources or references to expertly note them if they are only mentioned sporadically throughout the presentation. It is not reasonable to expect them to do so when our curriculum is designed specifically for external hires with no previous training or experience.
- Many of our VSR trainees are starting their career or are starting over and have not been in a formal education or training environment for a long time, sometimes many years. It only improves their ability to learn and training experience to organize the content in a more straight-forward, up-front manner.
- It is my understanding the WYLTKM slide should really only contain additional references/resources not specifically utilized in the course, but still relevant to the topic discussed for additional research and reading anyway. Therefore, this is insufficient to provide a summary of the references actually used during the training.

Question - if we are in agreement to begin providing references in the introduction of the course, do we still need to identify and list references for each topic and/or slide on the slide notes/transcript documents? How can we work smarter, not harder, and still comply with ISD recommendations? We want to make lesson/module creation/updates as efficient as possible for staff since we (very) busy.

Please comment with additional points of consideration, concerns, suggestions or other feedback, etc. Mary would like this discussed/resolved at our next meeting if possible.

**Comment by GizmoGomez** [comment]
I concur with Joanne's well-phrased suggestion. I have also received those comments from ISD and echo the sentiment about entry-level, external RVSRs. I find it helpful to give a "sneak peek" or quick overview of references at the beginning of the lesson as well as throughout then finishing with a list of other references to explore.

Thank you.

**Comment by CAGra8296** [comment]
I concur with Joanne,

Your recommendations are solid and I agree the suggestion given by ISD. I think we need to provide the references at the beginning and any additional ones to the WYLTKM (I probably messed that up).  I think it is good to have the references through the section level (example😊 M21-1 III.ii.1.A. – Requesting Evidence from Federal Record Custodians) just on the references in the lesson at the front and the WYLTKM.

I think it helps our instructors and backups to have the references for that particular slide on that slide notes. I think we need to go the next level down (topic level), so down lower than the other references. If it comes to making corrections for any additional changes of those references, we should be able to just focus on the slide notes themselves and not the lesson portion.

**Comment by kdavisva** [comment]
ISD recommended to ADD a slide after the objective that includes references and standards. I agree with this one.


Although the CCG recommends the references at the end of the module, when we were updating the courses for VIP, we had permission to include the reference slide after the objective slide in the module.  During that time, we never received a comment from ISD concerning the placement of the reference slide. It was our thought that because these VIP modules were for new employees it made more sense to have the references at the beginning vice the end. Plus, there were several instances where the references were needed to clarify a point during the module. 

As for the point to identify and list references for each topic and/or slide on the slide notes/transcript documents. I think that is overkill. Employee MUST learn how to navigate the manual. The more practice they get the better they will be. This need to constantly trying to provide everything to them does not help in the learning process. As for having the references on the slide notes for the instructors - isn't that part of the instructors preparation for the class? I would double check them anyway just to make sure there wasn't any updates. 

I think what needs to be remembered is that the CCG is a guide. Modification can be made to fit the scenario. In the case of VSR VIP modules, it would be better suited to have the references at the beginning vice the end. In addition,  I do not believe we need to include the WYLTKM side at the end. That is just duplication of effort.

**Comment by jrvictor** [comment]
@CAGra8296 great points about the slide/topic-level references being helpful for instructors and trainees for prep and delivery and traceability. If I'm not mistaken, I believe ISD wanted us to do this as well, so I'm not sure @kdavisva if we can remove that requirement, which is why I was asking about it because I also agree it feels redundant and cumbersome when updating things.

**Comment by jrvictor** [comment]
@CAGra8296 I agree with you 100%, that was also my experience instructing last session, filling in for courses last minute I wasn't even assigned as back-up for and I also didn't have sufficient time in the day to even prepare to lead the things I _was_ assigned back-up for either. Having the references ready-to-go is a huge time-saver and ensures we are providing sufficient/accurate information to whomever is instructing (no excuses if they provide incorrect info lol)

I also agree with you that it would be nice to just have one slide notes doc that is used for both instruction/trainees. I feel like giving trainees the information isn't really doing the work for them, it's helping since our chat-pods get out of control with these huge groups and they are overwhelmed trying to keep up with the instructor, the content and the references and questions in the chat, etc.

**Comment by jrvictor** [comment]
See comment in issue #40 regarding using endnotes for references instead of listing references on each document in transcript/slide notes.

https://github.com/compensationServiceTraining/curriculumCreationGuide/issues/40#issuecomment-1011234136

**Comment by artlessdodger** [comment]
Adding to this — @jrvictor, point three is incorrect:

_It is my understanding the WYLTKM slide should really only contain additional references/resources not specifically utilized in the course, but still relevant to the topic discussed for additional research and reading anyway. Therefore, this is insufficient to provide a summary of the references actually used during the training._

The Would You Like to Know More slide should contain all applicable references for the course, to include references cited in context where they are useful.

**Comment by MJones3231** [comment]
I disagree. Trainees have the transcript (self-paced) and slide notes (instructor-led) documents and individual "drilled-down" references are to be listed on each of those slides, according to the CCG. The WYLKM slide is for the broad topic references. Why do they need a list of broad topic references right after objectives?
